### ISSUE TITLE
Homeassistant is reporting usage of soon to be removed functionality. Detected that custom integration 'hacs' accesses hass.components.frontend. This is deprecated and will stop working in Home Assistant 2024.9

### ISSUE THREAD

**Issue by nmeachen** [hacs/integration#3946]
## System Information

version | core-2024.8.0
-- | --
installation_type | Home Assistant Container
dev | false
hassio | false
docker | true
user | root
virtualenv | false
python_version | 3.12.4
os_name | Linux
os_version | 6.6.28+rpt-rpi-v8
arch | aarch64
timezone | America/New_York
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 5000
Installed Version | 1.34.0
Stage | running
Available Repositories | 1391
Downloaded Repositories | 12
HACS Data | ok

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Dashboards</summary>

dashboards | 7
-- | --
resources | 7
views | 7
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | July 27, 2024 at 18:20
-- | --
current_recorder_run | August 7, 2024 at 20:18
estimated_db_size | 572.08 MiB
database_engine | sqlite
database_version | 3.45.3

</details>

### Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/issues>
- [X] This issue is related to the backend (integration part) of HACS.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This is a bug and not a feature request.
- [X] This issue is not a duplicate issue of currently [open](https://github.com/hacs/integration/issues) or issues [pending release](https://github.com/hacs/integration/issues?q=is%3Aissue+is%3Aclosed+sort%3Aupdated-desc+milestone%3Anext).

### Describe the issue

Homeassistant logs are reporting that the way hass.components.frontend are used will stop working in the 2024.9 release.

### Reproduction steps

1. Restart Homeassistant 2024.8
2. Read logs

### Debug logs

```text
Debug log not relevant for this issue. Here is the HomeAssistant log entry instead.

2024-08-07 20:18:16.184 WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'hacs' accesses hass.components.frontend. This is deprecated and will stop working in Home Assistant 2024.9, it should be updated to import functions used from frontend directly at custom_components/hacs/frontend.py, line 68: hass.components.frontend.async_register_built_in_panel(, please create a bug report at https://github.com/hacs/integration/issues
```

**Comment by neckyaan** [comment]
Detected that custom integration 'hacs' calls hass.http.register_static_path which is deprecated because it does blocking I/O in the event loop, instead call `await hass.http.async_register_static_paths([StaticPathConfig("/hacsfiles/iconset.js", "/config/custom_components/hacs/iconset.js", True)])`; This function will be removed in 2025.7 at custom_components/hacs/frontend.py, line 57: hass.http.register_static_path(, please create a bug report at https://github.com/hacs/integration/issues
Detected that custom integration 'hacs' accesses hass.components.frontend. This is deprecated and will stop working in Home Assistant 2024.9, it should be updated to import functions used from frontend directly at custom_components/hacs/frontend.py, line 68: hass.components.frontend.async_register_built_in_panel(, please create a bug report at https://github.com/hacs/integration/issues
Detected that custom integration 'hacs' calls hass.http.register_static_path which is deprecated because it does blocking I/O in the event loop, instead call `await hass.http.async_register_static_paths([StaticPathConfig("/hacsfiles", "/config/www/community", True)])`; This function will be removed in 2025.7 at custom_components/hacs/base.py, line 1162: self.hass.http.register_static_path(, please create a bug report at https://github.com/hacs/integration/issues
Detected that custom integration 'iphonedetect' accesses hass.helpers.event. This is deprecated and will stop working in Home Assistant 2024.11, it should be updated to import functions used from event directly at custom_components/iphonedetect/device_tracker.py, line 124: hass.helpers.event.track_point_in_utc_time(, please report it to the author of the 'iphonedetect' custom integration
Detected that custom integration 'pid_controller' calls `async_track_state_change` instead of `async_track_state_change_event` which is deprecated and will be removed in Home Assistant 2025.5 at custom_components/pid_controller/sensor.py, line 831: async_track_state_change(self.hass, entity, sensor_state_listener), please create a bug report at https://github.com/soloam/ha-pid-controller/issues

**Comment by PaulAntonDeen** [comment]
And this one as of HA Core 2024.8.0:
Logger: homeassistant.util.json
Source: helpers/deprecation.py:197
First occurred: 10:23:26 (24 occurrences)
Last logged: 10:23:29
json_loads was called from hacs, this is a deprecated function which will be removed in HA Core 2025.8. Use homeassistant.util.json.json_loads instead, please create a bug report at https://github.com/hacs/integration/issues

**Comment by tjorim** [comment]
Duplicate...

**Comment by nmeachen** [comment]
> Duplicate...

There is no other issue logged for async_register_built_in_panel, so this does not appear to be a duplicate. Others trying to hijack this issue are indeed duplicating other issues.

**Comment by nmeachen** [comment]
I stand corrected, this is a duplicate of #3534. I trust a new HACS package will be released this month.
